requirements/edx/kernel.in

```diff
@@ -115,7 +115,7 @@ openedx-calc                        # Library supporting mathematical calculatio
 openedx-django-require
 openedx-events>=8.3.0               # Open edX Events from Hooks Extension Framework (OEP-50)
 openedx-filters                     # Open edX Filters from Hooks Extension Framework (OEP-50)
-openedx-learning<=0.1
+git+https://github.com/open-craft/openedx-learning.git@chris/system-defined-taxonomies#egg=openedx-learning
```

Comment:
TODO: Update this when https://github.com/openedx/openedx-learning/pull/67 is deployed

Reply:
Once https://github.com/openedx/openedx-learning/pull/67#issuecomment-1659802119 is done, could you update this to be:

```
openedx-learning<0.2
```

We'll do all our MVP changes under 0.1.x, and so this will mean they're picked up in the platform automatically.

Is that ok @bradenmacdonald , or do we need to bump the minor release every time too?

Reply:
@bradenmacdonald  Ah cool, I'm happy with unpinned.

Reply:
@ChrisChV  I've merged your https://github.com/openedx/openedx-learning/pull/67 and tagged it as `v0.1.1`.

So if you unpin `openedx-learning` and re-compile the requirements, it should pick up the latest version?